layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationElementDrawer.vue

```diff
@@ -118,6 +126,9 @@ export default {
       this.selectedPage = null;
       this.$root.$emit('reset-element-drawer');
     },
+    expandDrawer() {
+      this.$refs.siteNavigationAddElementDrawer.expand = true;
```

Comment:
Please change it into a prop on `siteNavigationAddElementDrawer`
```suggestion
      this.expand = true;
```

Reply:
removed